### PR TITLE
toggle: Add toggle component to react layer

### PR DESCRIPTION
**Pull request by suabochica**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.idea
```
Comment (suabochica) [inline .gitignore:17]: 🤔 this looks weird? if we have a reason to keep it please share the explanation. If not, please remove this line 😄

File: src/Toggle/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Toggle
```
Comment (suabochica) [inline src/Toggle/README.md:1]: Please complete the README.md template: The idea here is explain the props that are part of the component and how we should use them. Keep in mind that this info is used in the Docs tab of storybook.

File: .gitignore (hunk 1 above)
Comment (suabochica) [reply .gitignore:17]: Forget it, I remember that you are using webstorm as IDE, so ignored this comment.

File: src/Toggle/README.md (hunk 2 above)
Comment (Burnedduk3) [reply src/Toggle/README.md:1]: sure, I'll do it today :D

File: src/Toggle/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
```
Comment (suabochica) [inline src/Toggle/README.md:1]: Sweet!

**Review by suabochica — COMMENTED** [review]
LGTM!

**Pull request merged**